### ISSUE TITLE
Method to check if a coords is a bin edge coord

### ISSUE THREAD

**Issue by arm61** [scipp/scipp#2467]
It would be nice to be able to call a method/property that returns a bool, `True` if the coord is a bin edge type.

```
print(my_dataarray.coords['my_coord'].is_bin_edge)
>> True
```

**Comment by jl-wynen** [comment]
The for you suggest is unfortunately not possible. `my_dataarray.coords['my_coord']` returns a `Variable` which does not know that it is part of a data array. So the API would have to be along the lines of
```.py
is_bin_edge(coord, data_array)
# or
is_bin_edge('coord_name', data_array)
```

**Comment by arm61** [comment]
this would also be fine by me 😎

**Comment by SimonHeybrock** [comment]
Note that the syntax @jl-wynen shows is in general not sufficient, since we also need to provide a dim-label.

For now I suggest to use this snippet for the check:

```python
da.coords['x'].sizes['x'] == da.sizes['x'] + 1
```

**Comment by jl-wynen** [comment]
Fixed by #2469
